NEWS.md

```diff
@@ -48,6 +48,10 @@
 * Added `stat_qq_line()` to make it easy to add a simple line to a Q-Q plot. This
   line makes it easier to judge the fit of the theoretical distribution 
   (@nicksolomon).
+  
+* Added `arrow.fill` parameter to `geom_segment()` and `geom_curve()` to 
```

Comment:
Can you please rewrite so the function names come first? (And then use that to alphabetise within this list of changes?)

(Normally I do that just before release but this release has been dragging on because I haven't found the time to finish it off)

Reply:
Done.